public/app/pipeline-orchestrator.js

```
@@ -215,6 +215,13 @@ class PipelineOrchestrator {
       console.warn('[Orchestrator] SharedArrayBuffer unavailable; live ML masking disabled:', err.message);
       this._inputRingSAB = null;
       this._maskRingSAB  = null;
+      try {
+        if (typeof globalThis !== 'undefined' && typeof globalThis.dispatchEvent === 'function' && typeof CustomEvent !== 'undefined') {
+          globalThis.dispatchEvent(new CustomEvent('ringAllocationFailed', {
```

ESLint

[error] 'CustomEvent' is not defined.

(no-undef)